content/learn/manipulating-files/node-writing-files.pt.md

```diff
@@ -0,0 +1,115 @@
+---
+title: writing-files-with-nodejs
+displayTitle: 'Escrevendo ficheiros com a Node.js'
+description: 'Como escrever ficheiros usando a Node.js'
+authors: flaviocopes, MylesBorins, fhemberger, LaRuaNa, ahmadawais, clean99, ovflowd, vaishnav-mk, nazarepiedady
+category: learn
+---
+
+## Escrevendo um ficheiro
+
+A maneira mais fácil de escrever em ficheiros na Node.js é usar a API `fs.writeFile()`:
+
+```js
+const fs = require('fs');
+
+const content = 'Some content!';
+
+fs.writeFile('/Users/joe/test.txt', content, err => {
+  if (err) {
+    console.error(err);
+  }
+  // ficheiro escrito com sucesso
+});
+```
+
+### Escrevendo um ficheiro de maneira síncrona
+
+De maneira alternativa, podes usar a versão síncrona `fs.writeFileSync()`:
+
+```js
+const fs = require('fs');
+
+const content = 'Some content!';
+
+try {
+  fs.writeFileSync('/Users/joe/test.txt', content);
+  // ficheiro escrito com sucesso
+} catch (err) {
+  console.error(err);
+}
+```
+
+Tu podes também usar o método `fsPromises.writeFile()` baseado em promessas oferecido pelo módulo `fs/promises`:
+
+```js
+const fs = require('fs/promises');
+
+async function example() {
+  try {
+    const content = 'Some content!';
+    await fs.writeFile('/Users/joe/test.txt', content);
+  } catch (err) {
+    console.log(err);
+  }
+}
+example();
+```
+
+Por padrão, esta API **substituirá os conteúdos do ficheiro** se o mesmo já existir.
+
+**Tu podes modificar o padrão especificando uma opção:**
+
+```js
+fs.writeFile('/Users/joe/test.txt', content, { flag: 'a+' }, err => {});
+```
+
+#### As opções que provavelmente usarás são
+
+| Opção | Descrição | Ficheiro é criado se não existir           |
+|------|-------------|:-----------------------------------------------:|
+| `r+` | Esta opção abre o ficheiro para **leitura** e **escrita** | ❌ |
+| `w+` | Esta opção abre o ficheiro para **leitura** e **escrita** e também posiciona a linha no **princípio** do ficheiro | ✅                                                          |
```

Comment:
```suggestion
| `w+` | Esta opção abre o ficheiro para **leitura** e **escrita** e também posiciona a linha no **ínicio** do ficheiro | ✅                                                          |
```

Reply:
I do not see the necessity of changing it, and since there are no grammar or syntactical errors, it should remain as it is.